com.unity.formats.alembic/Documentation~/ref_CustomData.md

```diff
@@ -0,0 +1,15 @@
+# Alembic Custom Data component
+
+The Alembic Custom Data component allows you to see additional custom information stored with each Face Set. A Face Set is a sub-group of faces that compose an object.
```

Comment:
Suggest: 
he Alembic Custom Data component displays additional custom information stored with each Face Set. A Face Set is a sub-group of faces that compose an object.